index.html

```diff
@@ -5,11 +5,22 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <meta name="description" content="Play incredible Star Wars Quiz! Can you beat the others? CLICK TO PLAY!">
+  <title>Star Wars Quiz</title>
+  <link rel="stylesheet" href="styles/index.css" />
 </head>
 
 <body>
-  <div id="app"></div>
+  <div id="#app" class='main-bg-image'>
```

Comment:
Now that the content of a page cannot be wider than 1440px, it would probably better if it was always centered. Please consider adding `display: flex` and `justify-content: center` to the styling of `main-bg-image` or giving them to the styling of `#app`.